chore: internal docs for beta (#187)
storyblok/storyblok-cli: src/commands/user/README.md

# User Command

> The `user` command allows you to view information about your currently logged-in Storyblok account.

Reviewer: ```suggestion
This command allows you to view information about your currently logged-in Storyblok account.
```
